apps/cookbook/src/app/examples/modal-example/modal-example-default.component.ts

```diff
@@ -1,28 +1,44 @@
 import { Component } from '@angular/core';
 
 import { ModalConfig, ModalController } from '@kirbydesign/designsystem';
+import { WindowRef } from '@kirbydesign/designsystem/types/window-ref';
 
-import { FirstEmbeddedModalExampleComponent } from './first-embedded-modal-example/first-embedded-modal-example.component';
 import { ModalCompactExampleComponent } from './compact-example/modal-compact-example.component';
-import { WindowRef } from '@kirbydesign/designsystem/types/window-ref';
+import { FirstEmbeddedModalExampleComponent } from './first-embedded-modal-example/first-embedded-modal-example.component';
 
 const config = {
   selector: 'cookbook-modal-example-default',
-  template: `<button kirby-button (click)="showModal()">Show modal</button>
+  template: `
+<button kirby-button (click)="showModal()">Show modal</button>
 <button kirby-button (click)="showDrawer()">Show drawer</button>
 <button kirby-button (click)="showCompact()">Show compact</button>
 <cookbook-example-configuration-wrapper>
-  <cookbook-modal-example-configuration
-    [(showDummyKeyboard)]="showDummyKeyboard"
+    <cookbook-modal-example-configuration [(showDummyKeyboard)]="showDummyKeyboard"
+    [(showPageProgress)]="showPageProgress"
     [(showFooter)]="showFooter"
     [(showDummyContent)]="showDummyContent"
     [(delayLoadDummyContent)]="delayLoadDummyContent"
     [(loadAdditionalContent)]="loadAdditionalContent"
-    [(openFullHeight)]="openFullHeight"
-  ></cookbook-modal-example-configuration>
-</cookbook-example-configuration-wrapper>`,
+    [(openFullHeight)]="openFullHeight">
+    </cookbook-modal-example-configuration>
+</cookbook-example-configuration-wrapper>
+    `,
   titleTemplate: `<kirby-page-title>My Modal Title</kirby-page-title>
  
+<p>Some content of the embedded component</p>
+...
+`,
+  pageProgressTemplate: `
+<kirby-page-header>
```

Comment:
Be aware of whitespace as this gets renderes in the code viewer:
```suggestion
  pageProgressTemplate: `<kirby-page-header>
```